### ISSUE TITLE
Добавить страницу "настройки" для пользователя

### ISSUE THREAD

**Issue by fey** [Hexlet/hexlet-friends#392]
Проблема: сейчас у пользователя нет раздела, куда можно было бы добавить настройки.

Как решим: 

* Добавляем Настройки в меню
![image](https://github.com/Hexlet/hexlet-friends/assets/697178/8407d045-fa2d-4c98-838d-b01e73ac9999)

* Добавляем раздел "Аккаунт", здесь добавляем раздел в котором пользователь сохраняет свой токен Github. Нам он потребуется позднее, чтобы синхронизировать данные пользователя с гитхабом

Верстку раздела можно подглядеть в СИКПе, либо в Хекслете
https://sicp.hexlet.io/ru/settings/account
https://ru.hexlet.io/account/settings/edit

**Comment by alllexxx1** [comment]
Беру в работу

**Comment by alllexxx1** [comment]
@fey, вы видите это таким образом или нужно что-то поменять?
![Screenshot from 2024-03-15 21-44-04](https://github.com/Hexlet/hexlet-friends/assets/120497608/62f210d5-7305-4d9a-8565-c7053f557d6e)
![Screenshot from 2024-03-15 21-44-04](https://github.com/Hexlet/hexlet-friends/assets/120497608/ebddf7e8-d166-402f-bd79-c545c20f62f9)

**Comment by fey** [comment]
ага, только где some token сразу инпут можно выводить с токеном.

**Comment by alllexxx1** [comment]
Вот так?
![image](https://github.com/Hexlet/hexlet-friends/assets/120497608/0fd88375-382d-4ed1-9d85-10101b6c2c49)

**Comment by fey** [comment]
@alllexxx1 да, верно. Туда должен подставляться будет токен пользователя. Если нет, то поле пустое.

**Comment by alllexxx1** [comment]
@fey , правильно я понимаю, что на данном этапе должна быть добавлена болванка формы, которая пока ничего не делает? 
Общий список изменений, которые я внес:
- добавил "настройки" в navbar;
- создал шаблон с настрйками "contributor_settings.html", вьюшку "contributor_settings.py";
-  в mixin.py добавил 2 миксина. Первый хэндлит неавторизованных пользователей, второй чужие настройки. Написал к ним документацию;
- сделал локализацю.
 Is it all right?

**Comment by fey** [comment]
@alllexxx1 можете добавить пользователю атрибут github_token чтобы форма могла уже сейчас добавлять токен юзеру.

Насчет того, что сделано - присылайте ПРчик и ссылку на демо =)

**Comment by alllexxx1** [comment]
@fey, поле github_token должно быть добавлено модели Contributor, верно?

**Comment by fey** [comment]
@fey если другой таблицы пользователя у нас нет, то да. 
Схема такая
1. пользователь логинится в hexlet-friends
2. идет в наастройки, ставит токен
3. используя этот токен потом тянутся данные с гитхаба у этого пользователя
